Description: remove google font from template.html, and link to local
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2015-09-09
Forwarded: not-needed
--- a/doc/template.html
+++ b/doc/template.html
@@ -4,10 +4,9 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
   <title>__SECTION__ | Node.js __VERSION__ Documentation</title>
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic">
   <link rel="stylesheet" href="assets/style.css">
   <link rel="stylesheet" href="assets/sh.css">
-  <link rel="canonical" href="https://nodejs.org/api/__FILENAME__.html">
+  <link rel="canonical" href="__FILENAME__.html">
 </head>
 <body class="alt apidoc" id="api-section-__FILENAME__">
   <div id="content" class="clearfix">
